apps/ide/src/plugins/project-wizard/build/build.js

```diff
@@ -14,11 +14,20 @@
  * limitations under the License.
  */
 
+/**
+ * @file Build runner
+ * @since 1.0.0
+ * @author cimfalab@gmail.com
```

Comment:
Hm, I thought that conventaion was applied for name.
I will consider change e-mail address.
Thank you~